interpreter/exec/simd.ml

```diff
@@ -97,6 +109,7 @@ sig
   module I64x2 : Int with type t = t and type lane = I64.t
   module F32x4 : Float with type t = t and type lane = F32.t
   module F64x2 : Float with type t = t and type lane = F64.t
+  module V128x1 : Vec with type t = t and type lane = I64.t
```

Comment:
Is there a reason for not sticking to the name used in the instructions and calling this plain V128?

Reply:
Because we have v128.ml, I thought that Simd.V128 might be a bit confusing.

Reply:
And V128x1 has symmetry with the rest of the module names here. I will switch it to V128 if you like, no strong preferences either way.

Reply:
Yes, I think V128 is preferable, since V128x1 is not a notion the language has, AFAIU.

Reply:
Sg, done!